tuxemon/resources/maps/bedroom_test.tmx

```diff
@@ -81,7 +81,7 @@
     <property name="act3" value="dialog_chain Go forth and inact my lampy will."/>
     <property name="act4" value="dialog_chain ${{end}}"/>
     <property name="cond1" value="is player_facing_tile"/>
-    <property name="cond2" value="is button_pressed K_RETURN"/>
+    <property name="cond2" value="is button_released K_RETURN"/>
```

Comment:
Dialogs should use "pressed" and not "released".  Using KEYUP/released events was a workaround events leaking between states.  Since this particular issue is fixed, these events should use KEYDOWN/pressed, as it provides instant pop up dialogs.